Sources/Basics/Observability.swift

```diff
@@ -321,10 +330,12 @@ public struct Diagnostic: CustomStringConvertible {
                 return 0
             case .info:
                 return 1
-            case .warning:
+            case .output:
                 return 2
-            case .error:
+            case .warning:
```

Comment:
Note, this numbering is internal and cannot affect clients, even though it shifts here.

Reply:
given the proposed number, output level would be filtered out from the default user facing output stream (>= warn). is that expected or is the idea output level is always "printed"